dags/databricks-automl-example.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
```

Comment:
```suggestion
from pendulum import datetime
```
This isn't a very publicized "standard" but `start_date` should _really_ be a `pendulum.datetime` object rather than a `datetime.datetime` object. This is only an absolute  for [time-zone-aware DAGs](https://airflow.apache.org/docs/apache-airflow/stable/timezone.html#time-zone-aware-dags) but IMO we should all start getting in the habit of using `pendulum.datetime` for `start_date` just in case users want to make our examples time zone aware.